src/main/java/com/jaquadro/minecraft/storagedrawers/client/renderer/ControllerRenderer.java

```diff
@@ -37,7 +39,7 @@ private void renderInventoryBlock(BlockController block, int metadata, int model
         GL11.glRotatef(90, 0, 1, 0);
         GL11.glTranslatef(-.5f, -.5f, -.5f);
 
-        RenderHelper.instance.state
+        RenderHelper.instances.get().state
```

Comment:
I'd cache the RenderHelper.instances.get() in a local variable if its used more than once.  It's cheap not but free.